apps/schools/domains/common/components/containers/BaseLayoutComponents/UserProfile/styles/styles.module.scss

```diff
@@ -5,15 +5,14 @@
 .menu {
   padding-top: 10px;
   border: 0;
-  display: flex;
   margin-top: auto;
 
   :global {
     .ant-menu-item-selected {
 
       color: $color-black;
       background-color: white !important;
-      font-family: "Roboto";
+      font-family: "Roboto", sans-serif;
       font-size: 16px;
       font-style: normal;
       font-weight: 600;
```

Comment:
^^
the same needs to be done here regarding fonts. Poor work with styles.

Reply:
https://lamart.atlassian.net/browse/SCHOOL-792